roofit/roofitcore/inc/RooAbsCollection.h

```diff
@@ -75,7 +80,10 @@ class RooAbsCollection : public TObject, public RooPrintable {
   virtual Bool_t remove(const RooAbsArg& var, Bool_t silent=kFALSE, Bool_t matchByNameOnly=kFALSE) ;
   virtual void removeAll() ;
 
-  virtual Bool_t add(const RooAbsCollection& list, Bool_t silent=kFALSE) ;
+  virtual Bool_t add(std::span<RooAbsArg * const> list, Bool_t silent=kFALSE) ;
+  virtual Bool_t add(const RooAbsCollection& list, Bool_t silent=kFALSE) {
```

Comment:
It really doesn't change anything, but if you want to modernise / use proper c++ types, it's:
```c++
virtual bool add(std::span<RooAbsArg * const> list, Bool_t silent=kFALSE) ;
virtual bool add(const RooAbsCollection& list, Bool_t silent=kFALSE) {
```
I only did it when I'm touching the code anyway, because it doesn't change anything in the program ...

Reply:
Done